package/CHANGELOG

```diff
@@ -20,6 +20,11 @@ The rules for this file:
 
 Enhancements
 
+  * Added a wrapper of lib.nsgrid in lib.distances.self_capped_distance
```

Comment:
@seb-buch also needs to be added as an author as he's authored some of the commits